omc3/response_creator.py

```diff
@@ -0,0 +1,87 @@
+"""
+Provides a response generation wrapper.
+The response matrices can be either created by response_madx or analytically via TwissResponse.
+
+:author: Joschua Dillys
+"""
+import os
+import pickle
```

Comment:
any updates on `feather` or `parquette`? pickle is problematic because it often requires the same packes to be installed (with compatible versions) as when the object was pickled.

Reply:
I would merge as is and later on make a decision for this, in alignment with the decision made in the ABP Computing group, so that we're all consistent. Considering we're pickling dictionaries sometimes, we could also look at `hdf`.